packages/gatsby-remark-images/src/index.js

```diff
@@ -71,11 +70,11 @@ module.exports = (
   )
 
   const getImageInfo = uri => {
-    const { url, query } = queryString.parseUrl(uri)
+    const url = new URL(uri)
+    url.search = '';
     return {
-      ext: path.extname(url).split(`.`).pop(),
-      url,
-      query,
```

Comment:
I dropped the `query` here because it's not used anywhere and the function isn't exported